astropy/utils/tests/test_misc.py

```diff
@@ -115,7 +114,7 @@ def test_check_broadcast():
 
 def test_dtype_bytes_or_chars():
     assert misc.dtype_bytes_or_chars(np.dtype(np.float64)) == 8
-    assert misc.dtype_bytes_or_chars(np.dtype(object)) == (8 if NUMPY_LT_1_10 else None)
+    assert misc.dtype_bytes_or_chars(np.dtype(object)) is None
```

Comment:
This will conflict with #6557. Not a real problem but the other PR is a Bugfix.

Reply:
Good point. I'll like then to hold back this PR until the other one is merged, so this can be rebased.